Arista/ninjaDump

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import sys
+import shutil
+import os
+
+from os import listdir
+from os.path import isfile, join
+
+ninjaStatsDirectory=os.environ.get('NINJA_STATS_DIRECTORY')
+filesToIngest = os.environ.get('NINJA_STATS_MAX_FILES_TO_INGEST')
+
+if not ninjaStatsDirectory:
+   print 'NINJA_STATS_DIRECTORY not configured'
+   sys.exit(1)
+
+# A safety valve to stop stats collection
+if isfile(join(ninjaStatsDirectory, "DISABLED")):
+   print 'Ninja stats collection disabled'
+   sys.exit(0)
+
+if not filesToIngest:
+   filesToIngest = '10'
+filesToIngest = int( filesToIngest )
+
+files = [ f for f in listdir( ninjaStatsDirectory ) if isfile(
```

Comment:
nit: files = sorted( [ .... ] )

So that the 10 files to be ingested would be in sort order.
Not sure if thats ok or if you need some other ordering.